src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadService.cs

```diff
@@ -70,6 +75,11 @@ public TrackedDownload TrackDownload(DownloadClientDefinition downloadClient, Do
                         return null;
                     }
 
+                    if (parsedEpisodeInfo == null)
```

Comment:
this `if` is a bit out of place, I think you should check for a `null` `parsedEpisodeInfo`as an `else` on line 67, you would have to re-arrange the method a bit but I think it would end up make more logical sense.